GO_stored_queries/GO_get_ancestors.yaml

```diff
@@ -0,0 +1,14 @@
+# Get all ancestors (all parent's parents) for this term
+
+name: GO_get_ancestors
+params:
+  type: object
+  required: [key]
+  properties:
```

Comment:
Let's add params here for `limit` and `offset` which can default to to 20 and 0 respectively. This way we 
In the query we'd add a line for `LIMIT @offset, @limit`
Let's also add a SORT BY clause in the query on the go term ID

Reply:
Also, in the json schema for the "offset" property, add a "maximum" of 10000